games-emulation/stella/stella-5.1.3.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit desktop gnome2-utils
+
+DESCRIPTION="Stella Atari 2600 VCS Emulator"
+HOMEPAGE="http://stella.sourceforge.net/"
+SRC_URI="https://github.com/stella-emu/stella/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
why not use the proper upstream release tarball `stella-5.1.3-src.tar.xz`?

Reply:
Because upstream moved from sourceforge to github.  Does github even provide this?

Reply:
yes it does, just take a look

Reply:
Never mind, I found it.

Just me being a github noob.